tokens/src/core/global/typography.json

```
@@ -11,7 +11,16 @@
         "sans": {
           "serif": {
             "source": "$font-family-sans-serif",
-            "$value": "-apple-system, BlinkMacSystemFont, 'Segoe UI',  Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji',  'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji'",
+            "$value": {
+              "primary": "-apple-system",
+              "secondary": "BlinkMacSystemFont",
+              "tertiary": "'Segoe UI'",
+              "alternate-1": "Roboto",
+              "alternate-2": "'Helvetica Neue'",
+              "alternate-3": "Arial",
+              "alternate-4": "'Noto Sans'",
+              "fallback": "sans-serif"
+            },
```

Based on the example [here](https://tr.designtokens.org/format/#example-19)

```suggestion
            "$value": ["-apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Helvetica Neue", "Arial", "Noto Sans", "sans-serif", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"],
```

---

Added in new commit [`2786dd4` (#3222)](https://github.com/openedx/paragon/pull/3222/commits/2786dd4056cd5e62ee5aae402788a7008d0ae8b4#diff-c8641349d8aa8a3b4b6654afc986d0d0d1b50d60899deb03e3b91c688e68fcd1L23)